common/utilities/string/wrap.cpp

```diff
@@ -0,0 +1,130 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include <string>
+#include <vector>
+#include <sstream>
+#include "wrap.h"
+#include "split.h"
+#include "../third-party/imgui/imgui.h"
```

Comment:
This might belong in something like <utilities/imgui/...>, what do you think? I don't like this include in here.

Reply:
Replaced with "#include <imgui.h>"

Reply:
We need to speak about it:
1. I use the original file , it is not under utilities.
2. I can change it to <imgui> and it works on the tools project,
The UT don't have an include to imgui so they don't compile.

The only way to fix it is to update the python script to look for key words to add project include path

Reply:
I'm suggesting that `utilities/string/wrap.h` be moved to `utilities/imgui/wrap.h`, that way the include makes more sense

Reply:
👍